src/chainstate/stacks/events.rs

```diff
@@ -15,6 +16,7 @@ pub use clarity::vm::events::StacksTransactionEvent;
 pub enum TransactionOrigin {
     Stacks(StacksTransaction),
     Burn(Txid),
+    NetworkProtocol,
```

Comment:
can you add comment explaining where this value comes from?

Reply:
It seems to refer to an event that originates from some internal bookkeeping, instead of from a burnchain or Stacks transaction.